src/Message/MerchantUrlsDataTrait.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
+
+use Omnipay\Common\Exception\InvalidRequestException;
+use Omnipay\Common\Exception\RuntimeException;
+
+trait MerchantUrlsDataTrait
+{
+    /**
+     * @return string
+     */
+    public function getAddressUpdateUrl()
+    {
+        return $this->getParameter('addressUpdateUrl');
+    }
+
+    /**
+     * @return string
+     */
+    public function getCancellationTermsUrl()
+    {
+        return $this->getParameter('cancellationTermsUrl');
+    }
+
+    /**
+     * @return array
+     *
+     * @throws InvalidRequestException
+     */
+    public function getMerchantUrls()
+    {
+        $this->validate('notifyUrl', 'returnUrl', 'termsUrl', 'validationUrl');
+
+        $merchantUrls = [
+            'checkout' => $this->getReturnUrl(),
```

Comment:
Adding an abstract function for `getReturnUrl`, `getNotifyUrl`, `getTermsUrl` and `getValidationUrl` would get Scrutinizer off our backs.

Reply:
For `getTermsUrl` there is a concrete implementation present, but agreed for `getReturnUrl` and `getNotifyUrl`